q2cli/tools.py

```diff
@@ -16,28 +16,32 @@ def tools():
     pass
 
 
-@tools.command(name='export', short_help='Export data.',
-               help="Export data from a QIIME Artifact or Visualization.")
-@click.option('--input-path', required=True,
-              type=click.Path(exists=True, dir_okay=True),
-              help='Path to Artifact or Visualization that should be '
-                   'exported.')
-@click.option('--output-path', required=True,
-              type=click.Path(exists=False, dir_okay=False),
-              help='Path where Result data should be written.')
-def export_data(input_path, output_path):
+@tools.command(name='export',
+               short_help='Export data from a QIIME Artifact or '
+                          'Visualization.',
+               help="Export data from a QIIME Artifact or Visualization. "
+                    "Exporting extracts the data stored in an Artifact or "
+                    "Visualization and will support exporting to multiple "
+                    "formats in the future. For now, the data is exported in "
+                    "the format it is stored within the Artifact or "
+                    "Visualization. Use 'qiime tools extract' to extract the "
+                    "Artifact or Visualization's entire archive.")
+@click.argument('path', type=click.Path(exists=True, dir_okay=False))
+@click.option('--output-dir', required=True,
+              type=click.Path(exists=False, dir_okay=True),
+              help='Directory where data should be exported to')
+def export_data(path, output_dir):
     import qiime.sdk
 
-    result = qiime.sdk.Result.load(input_path)
-
-    return result.export_data(output_path)
+    result = qiime.sdk.Result.load(path)
+    result.export_data(output_dir)
 
 
-@tools.command(name='import', short_help='Import data.',
+@tools.command(name='import', short_help='Import data into a QIIME Artifact.',
```

Comment:
Since we are mirroring `export` above, maybe we should clarify that this is a *new* QIIME Artifact? If someone is unfamiliar with QIIME 2 they might assume that you can import items into an existing artifact.

Options:
- `Import data into a new QIIME Artifact`
- `Create a new QIIME Artifact by importing data.`
- ???